db/migrate/20210317155738_create_apply_application_sync_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateApplyApplicationSyncRequests < ActiveRecord::Migration[6.1]
+  def change
+    create_table :apply_application_sync_requests do |t|
+      t.integer :response_code
```

Comment:
or `response_status`?

Reply:
Storing it as an integer I think `response_code` is better 👍